browser/ui/webui/brave_settings_ui.cc

```diff
@@ -129,6 +146,9 @@ void BraveSettingsUI::AddResources(content::WebUIDataSource* html_source,
                               net::features::kBraveForgetFirstPartyStorage));
   html_source->AddBoolean("isBraveRewardsSupported",
                           brave_rewards::IsSupportedForProfile(profile));
+  html_source->AddBoolean(
+      "areShortcutsSupported",
```

Comment:
I think it reads slightly nicer, `isShortcutsSuppported` feels slightly off. I think it's because we would say `Are shortcuts supported?` instead of `Is shortcuts supported?`. Brave rewards feels nicer with `is` though (and asking `Is brave rewards supported?` feels more natural).

Pretty nebulous though :laughing: